pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -113,6 +113,7 @@ func (db *DB) Migration() *migrate.Migration {
 						`ALTER TABLE bandwidth_agreements ADD COLUMN total INT(10)`,
 						`ALTER TABLE bandwidth_agreements ADD COLUMN max_size INT(10)`,
 						`ALTER TABLE bandwidth_agreements ADD COLUMN created_utc_sec INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN status TEXT DEFAULT 'UNSENT'`,
```

Comment:
This probably should be `int` and we can have enum for all values.

Reply:
updated with int values